symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +3029,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
```

Comment:
Would it be possible to define `hbar` in `sympy.physics` using `PyFunction`? That way I think, we could have a check for `USE_SYMENGINE=1` in the respective modules and define `hbar` if the condition is true?